rotkehlchen/tests/exchanges/test_kraken.py

```diff
@@ -884,3 +887,25 @@ def test_kraken_staking(rotkehlchen_api_server_with_exchanges, start_with_valid_
         },
     )
     assert_proper_response_with_result(response)
+
+    _, without_eth2_staking_report_result, _ = query_api_create_and_get_report(
+        server=rotkehlchen_api_server_with_exchanges,
+        start_ts=0,
+        end_ts=1640493374,
+        prepare_mocks=False,
+    )
+    without_eth2_staking_overview = without_eth2_staking_report_result['entries'][0]['overview']
+    assert FVal('39102.60').is_close(
+        FVal(without_eth2_staking_overview.get(str(AccountingEventType.STAKING))['taxable']),
+    )
+    rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen.data.db.set_settings(
+        ModifiableDBSettings(eth_staking_taxable_after_withdrawal_enabled=True),
+    )
+    _, with_eth2_staking_report_result, _ = query_api_create_and_get_report(
+        server=rotkehlchen_api_server_with_exchanges,
+        start_ts=0,
+        end_ts=1640493374,
+        prepare_mocks=False,
+    )
+    with_eth2_staking_overview = with_eth2_staking_report_result['entries'][0]['overview']
```

Comment:
Why is it empty? Shouldn't; XTZ staking also be taken into account? Or is it just a staking deposit?